Clinical-Genomics/scout
# Fix Mito Report

Bug fix #1352

**How to test**:
1. Make sure that mitochondrial report for case  https://scout-stage.scilifelab.se/cust002/F0009130-validation doesn't work. It triggers an error
1. Install this branch on stage and try to download report for the same case

**Expected outcome**:
Report should be downloadable and look fine

**Review:**
